db/migrate/20171214113834_delete_test_emails_and_delivery_attempts.rb

```diff
@@ -0,0 +1,7 @@
+class DeleteTestEmailsAndDeliveryAttempts < ActiveRecord::Migration[5.1]
+  def up
+    # Emails and DeliveryAttempts at this point are all test data
+    Email.destroy_all
+    DeliveryAttempt.destroy_all
```

Comment:
Think we might have to change the order of these statements... https://sentry.io/govuk/app-email-alert-api/issues/419031969/

Reply:
https://github.com/alphagov/email-alert-api/pull/327